src/js/components/modal/Zap.tsx

```diff
@@ -101,9 +102,10 @@ const Close = styled.div`
 export default function SendSats(props: ZapProps) {
   const onClose = props.onClose || (() => undefined);
   const { note, recipient } = props;
-  const defaultZapAmount = 1_000;
+  const defaultZapAmount = SocialNetwork.getDefaultZapAmount();
   const amounts = [defaultZapAmount, 5_000, 10_000, 20_000, 50_000, 100_000, 1_000_000];
   const emojis: Record<number, string> = {
+    [defaultZapAmount]: '⚡',
```

Comment:
I don't know why zap is not rendering here, but it worked well on the application running.